test/e2e/playbook/provision.yaml

```diff
@@ -290,12 +290,37 @@
 
     - name: Configure containerd
       when: is_containerd
-      ansible.builtin.shell: "{{ item }}"
-      with_items:
-        - mkdir -p /etc/containerd
-        - containerd config default > /etc/containerd/config.toml
-        - sed -i 's/^.*disabled_plugins *= *.*$/disabled_plugins = []/' /etc/containerd/config.toml
-        - sed -i 's/SystemdCgroup = false/SystemdCgroup = true/g' /etc/containerd/config.toml
+      block:
+        - name: Create containerd configuration
+          ansible.builtin.shell: "{{ item }}"
+          with_items:
+            - mkdir -p /etc/containerd
+            - containerd config default > /etc/containerd/config.toml
+            - sed -i 's/^.*disabled_plugins *= *.*$/disabled_plugins = []/' /etc/containerd/config.toml
+            - sed -i 's/SystemdCgroup = false/SystemdCgroup = true/g' /etc/containerd/config.toml
+
+        - name: Update CNI plugin directory on Fedora
+          when: ansible_facts['distribution'] == "Fedora"
+          ansible.builtin.shell: "{{ item }}"
+          with_items:
+            - sed -i 's:/opt/cni/bin:/usr/libexec/cni:g' /etc/containerd/config.toml
```

Comment:
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/lineinfile_module.html might be more efficient (faster than shell + sed)

Reply:
Updated to use `ansible.builtin.lineinfile`. Tested with `cni_plugin=bridge` and it still worked.